src/core/ext/filters/client_channel/lb_policy/outlier_detection/outlier_detection.h

```diff
@@ -30,7 +30,7 @@ namespace grpc_core {
 bool XdsOutlierDetectionEnabled();
 
 struct OutlierDetectionConfig {
-  Duration interval = Duration::Infinity();
+  Duration interval = Duration::Seconds(10);
```

Comment:
Was this a bug? Is the default 10 seconds?

Reply:
Yeah, the default is supposed to be 10s.  We were failing to default properly, and there was some logic previously to disable the policy if the interval was infinite, which we were triggering via this default.